main.tf

```diff
@@ -258,6 +258,7 @@ module "postgresql" {
   create_db_subnet_group    = true
   create_db_parameter_group = true
   create_db_option_group    = true
+  create_random_password    = false
```

Comment:
I did consider using an expression similar to what you suggested to conditionally enable creating a random password, however, since the create_random_password variable has been set to false in all terraform-aws-rds modules prior to 4.x and our module already provides a default administrator_password when a random pw is not generated, I chose the option of leaving create_random_password = false as before to preserve the existing behavior. If we conditionally set create_random_password, we will also need to update our doc to remove the default administrator_password value and document the new behavior which will add additional complexity to administrator password handling and testing that I wanted to minimize. Also, as you noticed for upcoming module releases, I was trying to minimize the churn to administrator_password behavior since consuming newer module updates is going to force password handling changes on us that is still possible to postpone with this update.